workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/container/LayeredDialogs.kt

```diff
@@ -34,7 +36,21 @@ public class LayeredDialogs(
   public constructor(
     view: View,
     modal: Boolean
-  ) : this(view.context, modal, { WorkflowLifecycleOwner.get(view) })
+  ) : this(
+    context = view.context,
+    modal = modal,
+    getParentLifecycleOwner = {
+      checkNotNull(WorkflowLifecycleOwner.get(view)) {
```

Comment:
It doesn't really have to be a `WorkflowLifecycleOwner`, use the androidx call instead.